testnets/hedgetestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "hedgetestnet",
+  "assets": [
+    {
+      "description": "The native staking token of the Hedge blockchain.",
+      "denom_units": [
+        {
+          "denom": "hedge",
+          "exponent": 0
+        },
+        {
+          "denom": "uhedge",
+          "exponent": 6
+        }
+      ],
+      "base": "hedge",
+      "name": "Berberis HEDGE",
+      "display": "hedge",
```

Comment:
I assume the token shouldn't be displayed in the base form. The display denom should match whichever you use for the exponent 6.